src/tools/debugger/entities/SteppingType.java

```diff
@@ -104,11 +110,37 @@ public void process(final Suspension susp) {
     }
   },
 
+  @SerializedName("stepToMessageRcvr")
+  STEP_TO_RECEIVER_MESSAGE("stepToMessageRcvr", "Step to Msg Receiver", Group.ACTOR_STEPPING, "msg-open", new Class[] {EventualMessageSend.class}, new ActivityType[] {ActivityType.ACTOR}) {
```

Comment:
I did a step-to-receiver-message in `<-:` and it works. Could you detail the scenario where does not work for me to reproduce?